src/utils/registration-helpers.ts

```diff
@@ -704,6 +704,17 @@ export const openFileInNewTab = (fileUri: string) => {
   }
 };
 
+const hasChangedContributors = (persistedContributors: Contributor[], updatedContributors: Contributor[]): boolean => {
+  if (persistedContributors.length !== updatedContributors.length) {
+    return true;
+  }
+
+  const sortedPersistedContributors = persistedContributors.map((contributor) => contributor.identity.id).sort();
+  const sortedUpdatedContributors = updatedContributors.map((contributor) => contributor.identity.id).sort();
```

Comment:
Dette vil også være en litt dyr operasjon når det er 5k contributors, siden det da må sorteres til sammen 10k elementer. veldig worst-case og ikke så realistisk da, men tipper det vil humpe litt før det blir 5k også. Lurer på om det kunne vært bedre å bare loope gjennom `persistedContributors` as is, og finne tilsvarende med `.find()` i `sortedUpdatedContributors`. Men om dette er presterer godt nok som det er er det sikkert greit nok

Reply:
`find()` vil være dyrere enn å bare slå opp indeksen direkte som du har gjort nå da, så godt mulig det vil prestere bedre uansett også, så bare se bort fra dette tror jeg